spec/datetimeformat.html

```diff
@@ -461,12 +458,12 @@ <h1>UnwrapDateTimeFormat( _dtf_ )</h1>
       </emu-alg>
       <emu-normative-optional>
       <emu-alg>
-        2. If _dtf_ does not have an [[InitializedDateTimeFormat]] internal slot and ? InstanceofOperator(_dtf_, %DateTimeFormat%) is *true*, then
+        1. If _dtf_ does not have an [[InitializedDateTimeFormat]] internal slot and ? InstanceofOperator(_dtf_, %DateTimeFormat%) is *true*, then
           1. Let _dtf_ be ? Get(_dtf_, %Intl%.[[FallbackSymbol]]).
       </emu-alg>
       </emu-normative-optional>
       <emu-alg>
-        3. Perform ? RequireInternalSlot(_dtf_, [[InitializedDateTimeFormat]]).
+        1. Perform ? RequireInternalSlot(_dtf_, [[InitializedDateTimeFormat]]).
```

Comment:
These changes break the rendering of https://tc39.es/ecma402/#sec-unwrapdatetimeformat . We must either keep the non-1 starting values or refactor the algorithms to no longer need them.